content/en/platform/corda/4.7/enterprise/node/operating/clientrpc.md

```diff
@@ -26,7 +26,7 @@ your node via HTTP, you will need to stand up your own webserver that connects t
 [here](https://github.com/corda/spring-webserver).
 {{< /warning >}}
```

Comment:
The links to the API docs don't resolve properly, they just go to the API homepage. 
Think you need to change the start of them from: 
https://api.corda.net/api/4.7/enterprise/html/api/javadoc/net/corda/client/rpc/ 
to:
https://docs.r3.com/en/api-ref/corda/4.7/enterprise/javadoc/net/corda/client/rpc